main.py

```diff
@@ -99,40 +107,45 @@ def reply(msg=None, img=None):
                     ('photo', 'image.jpg', img),
                 ])
             else:
-                logging.error('no msg or img specified')
+                logging.error('no msg or img specified') #If there is no image it puts in the google log the string
                 resp = None
 
             logging.info('send response:')
             logging.info(resp)
-
+        #From here you can make custom commands, just add an 'elif'
         if text.startswith('/'):
             if text == '/start':
                 reply('Bot enabled')
-                setEnabled(chat_id, True)
+                setEnabled(chat_id, True) #Sets the status to True (read above comments)
             elif text == '/stop':
                 reply('Bot disabled')
-                setEnabled(chat_id, False)
-            elif text == '/image':
-                img = Image.new('RGB', (512, 512))
+                setEnabled(chat_id, False) #Changes it to false
```

Comment:
The code is mostly self-explanatory. Only need to mention what is enabled or disabled (and that should be done elsewhere).